CMakeLists.txt

```diff
@@ -90,6 +90,20 @@ if(CERES_FOUND)
   include_directories(${CERES_INCLUDE_DIRS})
 endif(CERES_FOUND)
 
+find_package(HDF5 1.8 REQUIRED COMPONENTS CXX)
+
+if(HDF5_FOUND)
```

Comment:
With `REQUIRED` hdf5 is always found, so remove this conditional.

Reply:
I'm just following the same way that it was done for the other packages, see libxc, ceres, eigen above.

Reply:
Okay, I've gone ahead and removed the checks for hdf5 and the other libraries.